### DISCUSSION TITLE
Hide fileextension in nodes?

### DISCUSSION THREAD

**Discussion by pluberd** [comfyanonymous/ComfyUI#1818] [Ideas]
Is it possible to hide the fileextension in the modelloader-nodes? I think it would increase the general overview, cause you can make the nodes smaller without loosing the model name.

![grafik](https://github.com/comfyanonymous/ComfyUI/assets/113139382/83d21f82-eb03-4d2f-b31f-f6bac2c23e2d)

This is ugly ... only because the file extension ".safetensors" is so long.

**Comment by ardri1166** [comment]
Surprised that no one else is asking this. It's terrible when you try to squeeze the needed nodes on a 27" screen and still be able to read to read anything.

**Reply by pluberd** [reply]
Thx for opening the feature request.

**Comment by ltdrdata** [comment]
Please move issues related to front-end improvements to this location:
https://github.com/Comfy-Org/ComfyUI_frontend/issues
